### ISSUE TITLE
Rune Arrow Price

### ISSUE THREAD

**Issue by Warchas** [Open-RSC/Game#1655]
The price of rune arrows is doubled.

You are suppose to get 320 gold per arrow but you get 640 gold.

**Comment by Marwolf** [comment]
https://openrsc.com/itemdef/646

Shop Price: 800gp
Low Alch Price: 320gp
High Alch Price: 480gp

640 gold from selling to shop, perhaps?

**Comment by Warchas** [comment]
You get 

Released | 25 March 2002 (Update)
-- | --
Quest item | No
Members item | Yes
Tradeable | Yes
Equipable | No
Stackable | Yes
High alch | 480gp
Low alch | 320gp
Shop price | Not sold
Sale price | 320gp
Examine | Arrows with rune heads


Sale price should be the price you get if I am not mistaking?

**Comment by lmsv-mx123** [comment]
Yes and normally prices you get when selling something at the shop is the low alch price
